src/es/manhwases/src/eu/kanade/tachiyomi/extension/es/manhwases/ManhwasEs.kt

```diff
@@ -0,0 +1,21 @@
+package eu.kanade.tachiyomi.extension.es.manhwases
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import okhttp3.OkHttpClient
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ManhwasEs : Madara(
+    "Manhwas.es",
+    "https://manhwas.es",
+    "es",
+    dateFormat = SimpleDateFormat("MMM dd, yy", Locale("es")),
+) {
+
+    override val client: OkHttpClient = super.client.newBuilder()
+        .rateLimit(1, 2)
+        .build()
+
```

Comment:
why 1,2 ratelimit?

Reply:
I add this while testing, but not remember why. I saw that the page is calling a bunch of times to cf and I didn't want to spam it

Reply:
cf interceptor loads webview, so that spam won't be ratelimited by this